api/v1alpha1/oadp_types.go

```diff
@@ -306,6 +309,14 @@ func (dpa *DataProtectionApplication) BackupImages() bool {
 	return dpa.Spec.BackupImages == nil || *dpa.Spec.BackupImages
 }
 
+// Default DisableInformerCache behavior when nil to false
+func (dpa *DataProtectionApplication) DisableInformerCache() bool {
```

Comment:
would change function name to something like `GetDisableInformerCache`. Otherwise, seems this functions disables it instead of getting the bool value from it

Reply:
Done, updated